docs/uml-class-diagrams/middleware01/test/MiddlewareGisManagerImplementationTests/MiddlewareGisManagerImplementationTests.puml

```diff
@@ -1,9 +1,22 @@
 @startuml MiddlewareGisManagerImplementationTests
 
 package edu.oakland.test.middleware01 {
-    class MiddlewareGisManagerImplementationTests {
-
-    }
+  class MiddlewareGisManagerImplementationTests {
+    + void isGpsSignalStrengthReceived()
+    + void isLocationDataPointStored()
```

Comment:
You can't directly test a private method or attribute, so we've got to think about how this relates to the public interface. What if we did this as a test that the Gps Signal Received is the one that it stores, and one for that it does not try and store null Gps Signals if the strength was not high enough. I think that is your intention here, but maybe we can make the naming a little bit more clear with something like:
```suggestion
    + void locationDataPointStoredIsOneReceived()
    + void nullLocationDataPointNotStored()
```